pygfunction/pipes.py

```diff
@@ -3613,3 +3616,209 @@ def _Nusselt_number_turbulent_flow(Re, Pr, fDarcy):
     Nu = 0.125 * fDarcy * (Re - 1.0e3) * Pr / \
         (1.0 + 12.7 * np.sqrt(0.125*fDarcy) * (Pr**(2.0/3.0) - 1.0))
     return Nu
+
+
+def compute_R_fp(
+        pipe_type: PipeType, m_flow_borehole: float, r_in: Union[float, tuple, list],
+        r_out: Union[float, tuple, list], k_p: Union[float, tuple, list], epsilon: float, fluid: Fluid) -> float:
+    if pipe_type in [PipeType.SINGLE_UTUBE, PipeType.DOUBLE_UTUBE_SERIES, PipeType.DOUBLE_UTUBE_SERIES_ASYMMETRICAL]:
```

Comment:
Is there a difference between `PipeType.DOUBLE_UTUBE_SERIES` and `PipeType.DOUBLE_UTUBE_SERIES_ASYMMETRICAL`?

They seem to be treated the same way throughout the code. Do we need both?